view/loadView.js

```diff
@@ -151,7 +151,7 @@ function loadView(app, passport) {
       (req, res) => {
         const trackObj = {
           trackingId: viewConfig.trackingId,
-          user: JSON.stringify(req.user),
+          user: JSON.stringify(req.user).replace(/'/g,"apos;"),
```

Comment:
shouldn't it be `"&apos;"`?

Reply:
Can you add a comment about why you're not escaping it "normally" so people don't keep asking questions like this.